nhs/data/handling.py

```diff
@@ -106,39 +108,56 @@ def read_spreadsheets(
-                      census_code_col: str, abbreviation_column_name: str, long_column_name: str) -> dict[str, pl.LazyFrame | None]:
+def standardize_names(
+    df_dict: dict[str, pl.LazyFrame],
+    census_metadata: pl.LazyFrame,
+    census_code_col: str,
+    abbreviation_column_name: str,
+    long_column_name: str,
+) -> dict[str, pl.LazyFrame | None]:
     """
     Standardise the column names of a polar Lazy frame dictionary to make them more readable
 
-    This function will
-        1. Expand abbreviated names.
-        2. Converting to lower case.
-
-    If any parameters are None, this function returns prev_csv_pl_df without change. Keys in df_dict must have format 'G{int + letter}'.
-
+    This function will expand abbreviated names using a census metadata frame containing
+    mappings of abbreviated names to full names in the columns `abbreviation_column_name` and
+    `long_column_name`. The names are also converted to snake_case.
 
     Parameters:
     --------
     df_dict : dict[str, pl.LazyFrame]
-        A dictionary containing census data as value and filename as key or None. a dictionary of name-dataframe pair. Names must exist in the census_metadata[census_code_col]. All column names in the dataframes in df_dict with keys in format 'G{int + letter}'.
-    census_metadata : [pl.LazyFrame | None]
-        Census_metadata: Census metadata lazy frame with one standard sheet has a column called identification containing list of keys in the df_dict dictionary to standarise, and columns called abbreviated.... and long_column)names containing the abbreviated and long column names for data frames in the df_dict or None. If don't have abbreviated column, long column and census_code_col then df_dict won't be changed.
+        A dictionary containing census data as value and filename as key. Names
+        must exist in `census_metadata[census_code_col]`.
+    census_metadata : pl.LazyFrame
+        Lazy frame containing a column called `census_code_col` that contains
+        the keys in `df_dict` to standardise, and columns called `abbreviation_column_name`
+        and `long_column_name` containing the abbreviated and long column names
+        for frames in the `df_dict`.
     census_code_col : str
-        Identify different files in df_dict. Could not be None, name of the column in census_metadata that contains names in the df_dict to standarise. e.g. if df_dict have the pair "G03": and "G03" exist in the census_code_col column in metada, then the lazy frame will be standarised
+        Column name in `census_metadata` that contains the keys in `df_dict` to standardise..
+        e.g. if `df_dict` have the key `"G03"` and `"G03"` exist `census_metadata[census_code_col]`
+        then the lazy frame will be standarised.
     abbreviation_column_name : str
-        Column name of abbreviated names. Could not be None, short will be the default value.
```

Comment:
Don't need to write type information like `"Could not be None"`, this is clear to see in the type signature of `str`

Reply:
Saying what the default value is is fine, but wrap it in backticks \` so it displays as code block like so `"Long"`.

Reply:
'Long'